### PR TITLE
announce 4.102

### PR DESCRIPTION
**Pull request by jjergus**

### CODE REVIEW

**Review by fredemmott — APPROVED** [review]

File: _posts/2021-03-23-hhvm-4.102.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+HHVM 4.102 is released! This release marks the end of support for 4.96;
+HHVM 4.97&ndash;4.101 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+# Breaking Changes
+
+- Usage of `<<__Override>>` is now correctly enforced on *interfaces*
+  (previously it was only enforced on classes and traits). The typechecker will
+  raise an error if the attribute is used on a method that was not declared on
+  any of your interface's parent interfaces.
```
Comment (Wilfred) [inline _posts/2021-03-23-hhvm-4.102.markdown:16]: This doesn't sound right -- you can't use `__Override` when a class implements a method. You use it when a class method overrides another class/trait method, or when an interface method overrides another interface method.

Comment (jjergus) [reply _posts/2021-03-23-hhvm-4.102.markdown:16]: This is meant to describe what happens if you have `<<__Override>>` on a method *in an interface*. Previously it would be ignored, now it's a Hack error if the interface method with `<<__Override>>` doesn't actually override a method from some parent interface.

Methods with `<<__Override>>` that are not on interfaces (they're in a class/trait) are not affected, they behave like before (so probably like you described).

Should I rephrase this somehow to make that clearer?

Comment (Wilfred) [reply _posts/2021-03-23-hhvm-4.102.markdown:16]: Oh, gotcha. How about this wording?

> if the attribute is used on an interface method that was not declared on any of your parent interfaces.

Comment (jjergus) [reply _posts/2021-03-23-hhvm-4.102.markdown:16]: Thanks, fixed in https://github.com/hhvm/hhvm.com/commit/731207be1ce344a450691cf6480dd6a52eb57412
